VideoLocker/res/layout/row_course_list.xml

```diff
@@ -220,16 +175,35 @@
         </LinearLayout>
     </RelativeLayout>
 
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/course_certified_banner"
+        style="@style/bold_white_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_marginLeft="40dp"
+        android:layout_marginRight="40dp"
+        android:layout_marginTop="17dp"
+        android:background="@color/course_banner_blue"
+        android:gravity="center"
+        android:lineSpacingMultiplier="0.8"
+        android:padding="10dp"
```

Comment:
:heavy_check_mark: I guess, as the padding was probably only important in it's vertical application.